packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -23,6 +23,8 @@ const OUTPUT_PATHS = {
     getPaths().api.functions,
     getProject().isTypeScriptProject ? 'auth.ts' : 'auth.js'
   ),
+  service: path.join(getPaths().api.services, '/ethereumAuth/ethereumAuth.js'),
+  graphql: path.join(getPaths().api.graphql, 'ethereumAuth.sdl.js'),
```

Comment:
Hmm, this feels out of place here. Understood you'll need service and graphql path included, but hardcoding these for Ethereum Auth here probably isn't the right approach. Maybe modify files() so you can pass config from providers/etherum.js that effectively adds to the returned `files`? Something like this would be more extensible for any/all auth providers.

Any other (better) thoughts about this @dthyresson?

Reply:
Oh you weren't supposed to find that 🙃